app/models/member.rb

```diff
@@ -194,8 +194,8 @@ def self.search(query)
     phone_number = query.match(/^(?:\+\d{2}|00\d{2}|0)(\d{9})$/)
     return where("phone_number like ?", "%#{ phone_number[1] }") unless phone_number.nil?
 
-    # If query is blank, no need to filter. Default behaviour would be to return Member class, so we override by passing all
-    return where(:id => (Education.select(:member_id).where('status = 0').map(&:member_id) + Tag.select(:member_id).where(:name => Tag.active_by_tag).map(&:member_id))) if query.blank?
+    # return all active members if query is empty
+    return Member.active if query.blank?
```

Comment:
I think this is incorrect, the current implementation displays all studying AND active members.

Reply:
Hi Rick, below you'll see the code `Member.active` is referring to, I believe it's the same line of code. 

https://github.com/svsticky/constipated-koala/blob/69ab067096798328bb6c9ea44cd830f6147264a5/app/models/member.rb#L74-L83

Do you mean by active currently in a committee? I don't think there is a query for that yet

Reply:
> Do you mean by active currently in a committee? I don't think there is a query for that yet

Yeah, I though that it meant that (and maybe this method should be renamed to avoid confusion in the feature)
You're right, your change is indeed correct.